Sources/Logging/Strings/OfferingStrings.swift

```diff
@@ -155,6 +156,12 @@ extension OfferingStrings: LogMessage {
 
         case let .overriding_package(old, new):
             return "Package: \(old) already exists, overwriting with: \(new)"
+
+        case .known_issue_ios_18_4_simulator:
+            return "None of the products registered in the RevenueCat dashboard could be fetched from App Store " +
+            "Connect (or the StoreKit Configuration file if one is being used)." +
+            "\nThis issue is widely reported by iOS 18.4 simulator users. Try using a different simulator version." +
+            "\nMore information: https://github.com/RevenueCat/purchases-ios/issues/4954"
```

Comment:
might be a good idea to put a rev.cat link (we have credentials in 1pass for the link shortener) so that if we have a new, better place to point to in the future we can change that dynamically in the future